omrsigcompat/makefile

```diff
@@ -25,6 +25,20 @@ OBJECTS := omrsig
 OBJECTS := $(addsuffix $(OBJEXT),$(OBJECTS))
 EXPORT_FUNCTIONS_FILE := omrsig.exportlist
 
+ifeq (win,$(OMR_HOST_OS))
+COPYRIGHT_RC_FILE := ../../omrsigcompat/omrsig.rc
+COPYRIGHT_RES_FILE := omrsig.res
```

Comment:
So the order is:
omrsig.rc (already created at ../../omrsigcompat) ---------> omrsig.res (generated at omr/omrsigcompat) ------> omrsig.dll